codes.merritt.Nyrna.yml

```diff
@@ -32,17 +32,14 @@ modules:
     only-arches:
       - x86_64
     build-commands:
-      - cp -r packaging/linux/flatpak/build-flatpak.sh .
-      - "./build-flatpak.sh"
+      - ./build-flatpak.sh
     sources:
       - type: file
         url: https://github.com/Merrit/nyrna/releases/download/v2.9.2/Nyrna-Linux-Portable.tar.gz
         sha256: 4b905764e83fe96175adc0474ccced9c0d281190d3137ac774fb4dcbe0b6dff5
       - type: git
         url: https://github.com/Merrit/nyrna.git
-        tag: v2.9.2
-        # If flatpak-builder throws error `fatal: transport 'file' not allowed`
-        # during build resolve by running `git config --global --add protocol.file.allow always`
-        disable-shallow-clone: true
+        commit: d79731503b4ff831f9e9310c2dcea64b5f8279cc
+        branch: main
```

Comment:
you should NOT have a branch. This will break the build as soon as there is a new commit.

Reply:
sadly the doc is wrong here in the context of flathub. Also sadly the linter should have failed the build and I don't know it has been changed.

The goal is that build don't break, and this will because the commit will not match the branch.

Reply:
git url + `main` branch + commit hash should always be available, as long as my `main` branch isn't renamed or something, as far as I understand.

Reply:
no. `main` will change at every new commit. It will fail. It's not about the branch being available, it's about the branch "commit-ish" pointing somewhere else. Unlike a tag.

(a retag also cause failure)